utils.py

```diff
@@ -65,17 +65,31 @@ def rewriteEmail(oldDests, newDest):
 			:return:
 		"""
 		if isinstance( oldDests, list ):
-			return [rewriteEmail(x) for x in oldDests ]
+			return [rewriteEmail(x, newDest) for x in oldDests ]
 		else:
 			newAddress = oldDests.replace(".", "_dot_").replace("@", "_at_")
 			return "%s%s" % (newAddress, newDest)
 
 	if conf["viur.emailRecipientOverride"]:
 		logging.warning("Overriding destination %s with %s", dests, conf["viur.emailRecipientOverride"])
-		if conf["viur.emailRecipientOverride"].startswith("@"):
-			dests = rewriteEmail(dests, conf["viur.emailRecipientOverride"])
+
+		if isinstance(conf["viur.emailRecipientOverride"], list):
+			tmpDests = []
+			for newRecipient in conf["viur.emailRecipientOverride"]:
+				if newRecipient.startswith("@"):
+					# use extend here, because rewriteEmail returns a list if we have multiple oldDests
+					tmpDests.extend(rewriteEmail(dests, newRecipient))
+				else:
+					tmpDests.append(newRecipient)
+			dests = tmpDests
+
+		elif isinstance(conf["viur.emailRecipientOverride"], str):
```

Comment:
please check for basestring rather than str to match both str and unicode

Reply:
sounds good